android/filament-android/src/main/java/com/google/android/filament/Stream.java

```diff
@@ -55,10 +74,12 @@ public Builder() {
         }
 
         /**
-         * Creates a native stream. Native streams can sample data directly from an
+         * Creates a NATIVE stream. Native streams can sample data directly from an
          * opaque platform object such as a {@link android.graphics.SurfaceTexture SurfaceTexture}
          * on Android.
          *
+         * @deprecated This lacks synchronization so ACQUIRED is preferred.
```

Comment:
I don't think we should deprecate this one. It works with all versions of android, and there are some cases where synchronizing is not required, e.g. streaming a movie on a texture.